gui/storage/views.py

```diff
@@ -763,7 +763,7 @@ def zpool_scrub(request, vid):
             _('Pool output could not be parsed. Is the pool imported?')
         )
     if request.method == "POST":
-        if request.POST.get("scrub") == 'IN_PROGRESS':
+        if request.POST.get("scrub") in ['IN_PROGRESS', 'PAUSED']:
```

Comment:
@themylogin If a scrub is paused we cant restart it?

Reply:
@william-gr you want "Unpause" button available as well?

Reply:
@themylogin since we are already handling it, I dont see why we cant officially support pause/unpause. I understand this is out of the scope of the original ticket but do you see any reasons we shouldnt?